addons/base_vat/models/res_partner.py

```diff
@@ -67,9 +67,14 @@
     'se': 'SE123456789701',
     'si': 'SI12345679',
     'sk': 'SK0012345675',
-    'tr': 'TR1234567890 (VERGINO) veya TR12345678901 (TCKIMLIKNO)'  # Levent Karakas @ Eska Yazilim A.S.
+    'tr': 'TR1234567890 (VERGINO) veya TR12345678901 (TCKIMLIKNO)',  # Levent Karakas @ Eska Yazilim A.S.
+    'xi': 'XI123456782',
 }
 
+_known_country_codes = [
+    'xi',
+]
```

Comment:
Rather use a set, just in case we add other specific codes like that in the future.